packages/generator/src/generator.js

```diff
@@ -83,6 +83,11 @@ export default class Generator {
       this.options.router.mode === 'hash'
         ? ['/']
         : flatRoutes(this.options.router.routes)
+
+    this.options.generate.exclude.forEach((regex) => {
+      routes = routes.filter(route => !regex.test(route))
```

Comment:
each .filter` creates a clone of the array. So can we use something like `filter(route =>  this.options.generate.exclud.some()` ?